Dockerfile

```diff
@@ -1,16 +1,23 @@
 # 1. Base on Node + Alpine for minimal size
-FROM node:22-alpine
+FROM mcr.microsoft.com/dotnet/sdk:9.0

```

Comment:
https://learn.microsoft.com/en-us/aspnet/core/client-side/libman/libman-cli?view=aspnetcore-9.0#restore-library-files

    dotnet tool install -g Microsoft.Web.LibraryManager.Cli

Misschien heben we hier dan enkel de Runtime:9.0 nodig, kleinere container

Reply:
![image](https://github.com/user-attachments/assets/e75cbb31-2fd0-4c0b-8d26-2ae19f9add3d)

We kunnen niet zomaar de `dotnet/runtime` image als base gebruiken blijkbaar